rcswitch/webif/__init__.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# vim: set encoding=utf-8 tabstop=4 softtabstop=4 shiftwidth=4 expandtab
+#########################################################################
+#  Copyright 2020-     <AUTHOR>                                   <EMAIL>
+#########################################################################
+#  This file is part of SmartHomeNG.
+#  https://www.smarthomeNG.de
+#  https://knx-user-forum.de/forum/supportforen/smarthome-py
+#
+#  Sample plugin for new plugins to run with SmartHomeNG version 1.5 and
```

Comment:
Das ist doch kein Beispiel (sample) Plugin oder?

Reply:
erl. in PR für `rcs1000n`